src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -88,7 +168,8 @@ public void forbidRenewalLoanByIdWhenFirstRequestInQueueIsRecall() throws Interr
   }
 
   @Test
-  public void allowRenewalLoanByIdWhenFirstRequestInQueueIsHold() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+  public void allowRenewalLoanByIdWhenProfileIsRollingFirstRequestInQueueIsHoldAndRenewingIsAllowedInLoanPolicy()
```

Comment:
@dmytrotur Done.

Reply:
@bohdan-suprun I'm sorry for poorly expressing my idea. I meant to explicitly setup **loan policy** that allows **renewal for loans with a pending hold request** rather than to rely on default policy. I won't change test behavior but will help to better understand the test's goal.

Reply:
@dmytrotur, no prob, let me try to find out how to better implement it.

Reply:
@dmytrotur done